src/js/utils/dom.js

```diff
@@ -90,6 +90,9 @@ export function sanitizeScriptNodes(element) {
     return element;
 }
 
+// Source: https://owasp.org/www-community/OWASP_Validation_Regex_Repository
+const validUrl = new RegExp(/^((((https?|ftps?|gopher|telnet|nntp):\/\/)|(mailto:|news:))(%[0-9A-Fa-f]{2}|[-()_.!~*';/?:@&=+$,A-Za-z0-9])+)([).!';/?:,][[:blank:|:blank:]])?$/);
+
```

Comment:
This just validated the URL as a well-formed URL - could be from any source. I don't think this solves the issue.

Reply:
Also we don't need to allow `ftps?|gopher|telnet|nntp):\/\/)|(mailto:|news` URLS for logo and ad click-throughs.

Reply:
@robwalch makes sense. I removed the unsupported protocols.